src/main/java/core/basesyntax/model/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Rectangle extends Color {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double width, double length) {
+        super(color);
+        this.width = width;
+        this.length = length;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The method draw() should not be in the same class as getArea(). You should separate the responsibilities into different interfaces.